examples/egui/Cargo.toml

```diff
@@ -34,3 +34,12 @@ rend3-framework = { version = "^0.3.0", path = "../../rend3-framework" }
 rend3-routine = { version = "^0.3.0",path = "../../rend3-routine" }
 # Windowing
 winit = "0.26"
+# Opening URL's
+webbrowser = "0.5.5"
+# Creating textures
+wgpu = "0.12"
+
+[dependencies.image]
```

Comment:
Lets keep this in the `image { version = "0.23", default-features = false, features = ["png"] }` form just to be consistent. Make sure to put this new line in alphabetical order